tests/trainer/legacy_deprecate_flow_log_tests/test_trainer_steps_scalar_return.py

```diff
@@ -190,3 +192,33 @@ def test_train_step_epoch_end_scalar(tmpdir):
     opt_closure_result = trainer.train_loop.training_step_and_backward(
         batch, batch_idx, 0, trainer.optimizers[0], trainer.hiddens)
     assert opt_closure_result['loss'].item() == 171
+
+
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+def test_dpp_reduce_mean_pbar(tmpdir):
+    class ExtentedModel(BoringModel):
+
+        logged = []
+
+        def training_step(self, batch, batch_idx):
+            output = self.layer(batch)
+            loss = self.loss(batch, output)
+            loss /= loss.clone().detach()
```

Comment:
```suggestion
            loss /= loss.clone().detach()
            self.log('self_log', loss, sync_dist=True)
```
Should reproduce the bug now :]